README.md

```diff
@@ -113,8 +118,10 @@ See the [LICENSE][license_file] file for the full license.
 
 [license_file]: https://github.com/CycloneDX/cyclonedx-python/blob/master/LICENSE
 [contributing_file]: https://github.com/CycloneDX/cyclonedx-python/blob/master/CONTRIBUTING.md
+[link_rtfd]: https://cyclonedx-bom-tool.readthedocs.io/
 
 [shield_gh-workflow-test]: https://img.shields.io/github/workflow/status/CycloneDX/cyclonedx-python/Python%20CI/master?logo=GitHub&logoColor=white "build"
+[shield_rtfd]: https://img.shields.io/readthedocs/cyclonedx-bom-too
```

Comment:
used a badge from shields.io, instead of the one from RTFD themselfs,
because shoields does not use cookies, claims to not track users, etc ..
abd because it fits the theme